### DISCUSSION TITLE
Is it `Yazi v1.0.6` or `Yazi v0.1.6`?

### DISCUSSION THREAD

**Discussion by azzamsa** [sxyazi/yazi#294] [1. Q&A] [answered]
Hi

```
WARNING: `display_name` is deprecated and will be removed in Yazi v1.0.6. Use `desc` instead.
e.g. { exec = 'nvim "$@"', display_name = "Edit" }  ==>  { exec = 'nvim "$@"', desc = "Edit" }
```

I think you mean `v0.1.6`?

**Comment by sxyazi** [comment, marked as answer]
Ah! You're right. That's so embarrassing, I actually wrote the wrong version 🤣

I've fixed it and postponed it to v0.1.7. Sorry for the confusion. https://github.com/sxyazi/yazi/commit/df4b502808fe994815073e2face26e22ceb479bc
